BlueprintUI/Sources/Layout/Aligned.swift

```diff
@@ -14,6 +14,9 @@ public struct Aligned: Element {
         case center
         /// Aligns the content to the bottom edge of the containing element.
         case bottom
+        
+        /// The content fills the full vertical height of the containing element.
+        case fill
```

Comment:
We should update the docs on this type if we add this, to something like:

> **For alignments other than `fill`,** the size of the content element is determined by calling `measure(in:)` on the content element – even if that size is larger than the wrapping element.

Reply:
Done